sonar-core/src/main/java/org/sonar/core/issue/DefaultIssue.java

```diff
@@ -265,8 +265,14 @@ public Double effortToFix() {
   }
 
   public DefaultIssue setEffortToFix(@Nullable Double d) {
-    Preconditions.checkArgument(d == null || d >= 0, "Effort to fix must be greater than or equal 0 (got " + d + ")");
-    this.effortToFix = d;
+    //Preconditions.checkArgument(d == null || d >= 0, "Effort to fix must be greater than or equal 0 (got " + d + ")");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)